### PR TITLE
 Custom Resnet9 || CIFAR10 || Apple

### PR DESCRIPTION
**Pull request by tiagoakle**

### CODE REVIEW

**Comment by tiagoakle** [comment]
Hi @codyaustun anything I can do to help?

File: CIFAR10/train/apple_resnet9_8v100_pytorch.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "version": "v1.0",
+    "author": "Santiago Akle Serrano, Hadi Pour Ansari, Vipul Gupta, Dennis DeCoste",
+    "authorEmail": "sakle@apple.com",
+    "framework": "Pytorch 1.1.0",
+    "codeURL": "https://github.com/apple/ml-cifar-10-faster",
+    "model": "Custom Resnet 9",
+    "hardware": "Tesla V100 * 8 GPU / 32 GB / 40 CPU",
+    "timestamp": "2019-12-11",
+    "misc": {}
```
Comment (codyaustun) [inline CIFAR10/train/apple_resnet9_8v100_pytorch.json]: Can you add details about your training routine (e.g., learning rate schedule, batch size, optimizer, etc.) and versions for any other important libraries (e.g., Cuda)? For example, take a look at [David Page's original submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/master/CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json).

File: CIFAR10/train/apple_resnet9_8v100_pytorch.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "version": "v1.0",
+    "author": "Santiago Akle Serrano, Hadi Pour Ansari, Vipul Gupta, Dennis DeCoste",
+    "authorEmail": "sakle@apple.com",
+    "framework": "Pytorch 1.1.0",
+    "codeURL": "https://github.com/apple/ml-cifar-10-faster",
```
Comment (codyaustun) [inline CIFAR10/train/apple_resnet9_8v100_pytorch.json:6]: Please specify the commit hash either in the `codeURL` or as a separate `commitHash` field

**Review by codyaustun — CHANGES_REQUESTED** [review]

**Comment by codyaustun** [comment]
> Hi @codyaustun anything I can do to help?

Sorry for the delay! Overall the submission looks good, but it would be nice to add more information to the JSON file.

**Comment by tiagoakle** [comment]
I think this addresses the comments

**Review by codyaustun — APPROVED** [review]
Thanks! Looks good to me!
